tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -80,4 +81,8 @@ export * from '@testing-library/react';
  */
 const fireEvent = reactRtlFireEvent;
 
+export function mountGlobalModal(context) {
```

Comment:
since this function is not RTL related, should we move it to the file `sentry/tests/js/sentry-test/utils.tsx `?

Reply:
It usese rtl's mount though

Reply:
Yep. I intentionally added this `mountGlobalModal()` with `reactTestingLibrary.tsx` so that it's obvious it is the RTL variant when we refactor pre-existing React tests to RTL.

Reply:
sold it :)